aioshelly/block_device/device.py

```diff
@@ -285,7 +286,7 @@ async def http_request(
         try:
             resp: ClientResponse = await self.aiohttp_session.request(
                 method,
-                f"http://{self.options.ip_address}/{path}",
+                URL.build(scheme="http", host=self.options.ip_address, path=path),
```

Comment:
```suggestion
                URL.build(scheme="http", host=self.options.ip_address, path=f"/{path}"),
```

Reply:
You're correct it should have a "/", weird that it didn't trigger any issues in my setup.